src/render/src/lib.rs

```diff
@@ -143,3 +148,80 @@ pub mod pso;
 pub mod shade;
 /// Convenience macros
 pub mod macros;
+*/
+
+use std::ops;
+use std::sync::mpsc;
+
+pub struct Gpu<B: Backend> {
+    device: Device<B>,
+    queue: CommandQueue<B, core::General>,
```

Comment:
should be `General` or `Graphics`. OpenGL backend will only expose graphics queues if there is no compute support,
Else I think every backend exposes general but need to double-check.

Reply:
Would a third `Compute`-only alternative be useful for some use cases ?

Reply:
I would say barely